### ISSUE TITLE
Underlines are not provided for the links present in the footer of the page.:A11y_AzureResponsibleAIworkshop_Footer_UseOfColor

### ISSUE THREAD

**Issue by PoojaNamde** [Azure/responsible-ai-hub#52]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #WCP; #Benchmark; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #Azure Responsible AI workshop; #CelaP3_Azure Responsible AI workshop_Mar2024; #WCAG1.4.1; #UseOfColor;

### Environment Details:
Application Name: Azure Responsible AI workshop
URL: https://azure.github.io/responsible-ai-hub
Chromium Edge Version 124.0.2478.6 (Official build) dev (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: (https://azure.github.io/responsible-ai-hub/)
2. TAB to links present at footer of the page.
3. Observe the issue.

### Actual Results:
Underlines are not provided for the links present in the footer of the page.

### Expected Results:
Underlines should be provided for the links present in the footer of the page.

### User Impact:
Color blind users will not be able to understand whether those are the links or normal text.

### Attachment:
